include/deal.II/lac/la_parallel_vector.h

```diff
@@ -244,7 +244,7 @@ namespace LinearAlgebra
       /**
        * Destructor.
        */
-      virtual ~Vector ();
+      ~Vector () override;
```

Comment:
Why should the destructor be special? We just make sure that the base class destructor is `virtual`.